### PR TITLE
Plaid-2016 tonnerre write-up (crypto)

### PR DESCRIPTION
**Pull request by JesseEmond**
See [my fork](https://github.com/JesseEmond/write-ups/tree/plaid_tonnerre/plaidctf-2016/tonnerre) to see what it looks like.

Please let me know if you feel that some parts are not clear enough or could benefit from more/less details.

cc @DrPandemic @carapas @KertanLeGnome , anyone of you could review/merge this please? Thanks!

### CODE REVIEW

File: plaidctf-2016/tonnerre/README.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+Let's play with our current equation to produce a valid `c`:
+```
+           g ^ (N-1) = 1
+=>   g ^ (N-1) * g^2 = 1 * g^2     (* g^2 on each side)
+=>       g ^ (N-1+2) = g^2         (group exponents)
+=>         g ^ (N+1) = g^2
+```
+
+Now, `c = g^2` and is valid! We can produce `c` with:
```
Comment (DrPandemic) [inline plaidctf-2016/tonnerre/README.md]: This sentence may sound weird

**Comment by DrPandemic** [comment]
I like it,but the SQL injection at the beginning could be worth explaining. If you don't think the injection should be explained :+1:

**Comment by JesseEmond** [comment]
@DrPandemic updated, thanks for the comment (I did not notice the weird sentence).

As for the SQL injection, I agree, although I have no idea how it was done ( @carapas gave us the extracted values and we worked from there, he could make a PR after this gets merged if he wants to explain the SQLi part imo ).

**Comment by JesseEmond** [comment]
@DrPandemic ping

**Pull request merged**